jenkins/run

```diff
@@ -54,6 +54,9 @@ if [[ ${?} -ne 0 ]]; then
     exit 1
 fi
 
+find . -name __pycache__ -exec rm -r {} \; > /dev/null 2>&1
```

Comment:
should we do `-name __pycache__ -or -name '*.pyc'` I find sometime I need to clear .pyc files to get the pytest working, but if they also get clear when we clear the cache thats fine

Reply:
Where do you find the `.pyc` files?

Reply:
Ahh interesting I didnt notice they all fall under `__pycache__` directory, cool